docs/UserGuide.md

```diff
@@ -1,751 +1 @@
| **Add**             | `add n/NAME j/POSITION d/DEPARTMENT p/PHONE e/EMAIL a/ADDRESS l/LEAVE [t/TAG]…​` <br> e.g. `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate p/1234567 l/14 t/criminal` |
| **Edit**            | `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [j/POSITION] [d/DEPARTMENT] [a/ADDRESS] [l/LEAVE] [t/TAG]…​` <br> e.g. `edit 1 p/91234567 t/friend e/newemail@example.com`                  |
| **Delete**          | `delete INDEX`<br> e.g. `delete 3`                                                                                                                                                   |
| **Batch Add**       | `batch-add FILENAME` <br> e.g. `batch-add newemployees.csv`                                                                                                                          |
| **View Details**    | `view INDEX` <br> e.g. `view 1`                                                                                                                                                      |
| **List**            | `list`                                                                                                                                                                               |
| **Find**            | `find [n/NAME] [j/POSITION] [d/DEPARTMENT]`<br> e.g. `find n/John j/engineer d/Tech`                                                                                                 |
| **Add leave**       | `add-leave id/ID sd/START_DATE ed/END_DATE`<br> e.g. `add-leave id/1 sd/01-01-2022 ed/02-01-2022`                                                                                    |
| **Delete leave**    | `delete-leave id/ID i/INDEX`<br> e.g. `delete-leave id/1 i/3`                                                                                                                        |
| **Rate**            | `rate id/ID r/RATING`<br> e.g. `rate id/1 r/3`                                                                                                                                       |
| **View Department** | `view-department DEPARTMENT`<br> e.g. `view-department Finance`                                                                                                                      |
| **Help**            | `help`                                                                                                                                                                               |
| **Exit**            | `exit`                                                                                                                                                                               |
| **Clear**           | `clear`                                                                                                                                                                              |

```

Comment:
Sorry but what is this.

Reply:
how windows interpret a new line, the clrf thing

Reply:
rate command different branch different commit